packages/synapse-react-client/src/components/FeaturedDataTabs/FacetPlotsCard.tsx

```diff
@@ -113,73 +163,68 @@ const FacetPlotsCard: React.FunctionComponent<FacetPlotsCardProps> = ({
         setSelectedFacetValue(selectedFacet?.value)
       }
     }
-  }, [facetsToPlot, data])
+  }, [facetsToPlot, data, accessToken])
 
   if (
     isLoadingNewBundle ||
     !facetPlotDataArray ||
     !facetDataArray ||
     facetDataArray.length === 0
   ) {
-    return (
-      <div className="FacetPlotsCard FacetPlotsCard__loading SRC-centerContentColumn">
-        {loadingScreen}
-      </div>
-    )
+    return <LoadingCard numPlots={(facetsToPlot ?? []).length} />
```

Comment:
Add a generic loading card that fills the space properly using the grid layout. It's really nice to have a skeleton that closely match the actual result in this case especially, because if plots come from independent queries, they will finish at different times.